app/views/shared/_money_input.html.slim

```diff
@@ -1,7 +1,12 @@
-- size ||= nil
+ruby:
+  size ||= nil
+  set_aria ||= nil
```

Comment:
Super minor pedantry, but we treat `set_aria` as a boolean, so maybe it should default to `false` rather than `nil`. Makes _no_ practical difference, so feel free to ignore!